### ISSUE TITLE
[Provider Request][Nextgen]: Clinical Associates MD (Maryland)

### ISSUE THREAD

**Issue by AzzieDev** [fastenhealth/fasten-onprem#529]
### Provider/Institution Name

Clinical Associates MD

### Provider/Institution Website

https://clinicalassociatesmd.com/

### Patient Portal Login Website

https://pxpportal.nextgen.com/clinicalassociates-28483/portal/#/user/login

### Provider EHR Platform

NextGen

### Anything else we should know?

It sounds like this is just another NextGen site.

**Comment by AnalogJ** [comment]
I already see this institution in the catalog - "Clinical Associates"
Is that not working for you?

I'm going to close this issue, but feel free to reopen if you still need help.

**Comment by AnalogJ** [comment]
related https://github.com/fastenhealth/fasten-sources/issues/20

**Comment by AzzieDev** [comment]
@AnalogJ yes it's a different "Clinical Associates", the one you referred to is in Texas not Maryland.

Please reopen if this comment doesn't do that.

**Comment by AnalogJ** [comment]
Hey

Sorry, we're updating this repo to make it clear that Fasten Onprem is essentially unable to retrieve EHR records directly. 

see https://github.com/fastenhealth/fasten-onprem/issues/629

Nextgen is supported by Fasten Connect, but we were not able to roll that functionality into Lighthouse
